### PR TITLE
feature: gateways without billing fields provide note to user instead

### LINKED ISSUE #4670: feat: add text and graphic to gateways that do not have fields or are offsite in form template
# Feature Request

## User Story

As a donor, I want a clear understanding of the donation experience so that I'm not confused when a gateway takes me off site or there is a general lack of text.

As an admin, I want to know how the functionality of the Test Gateway works and don't want my donors to be confused when a gateway doesn't have any text that displays when clicking on it.

## Details

Currently right now with Sequoia some gateways have fields that display (Credit Card gateways) or text (Offline Donations) while others do not have any (PayPal Standard). This leads to a poor donor experience when selecting the different options. What we need to do is ensure each gateway has the appropriate text that informs the donor and is clear and concise.  

## Visuals

**PayPal Standard**

**Test Gateway**

**Offline Donations**

## Possible Solution

Each gateway that's offsite needs to have text that makes it clear to the user what to expect. We need to ensure that the text that displays is styled appropriately, has the light gray background, and has a nice graphic whenever possible.

## Acceptance Criteria

- [x] PayPal Standard has text and image that displays appropriately
- [ ] Razorpay has text and an image that displays appropriately
- [x] Test Gateway has text and an image that displays appropriately
- [x] Offline Donations has text and an image that displays appropriately 
- [x] The actual UI matches the concept for all gateways

### PR DESCRIPTION
**Pull request by henryholtgeerts**
## Description
Resolves #4670 
This PR extends existing payment gateway functionality to that gateways without biling fields provide a note about the gateway or potential offsite redirection instead.

This functionality is especially useful to the Sequoia template, where gateways are displayed in an accordion tab format, and empty gateways sections felt lacking.

_NOTE: This PR does not introduce a note for the Razorpay gateway. A separate issue has been addressed to introduce this functionality to the addon. See: https://github.com/impress-org/give-razorpay/issues/55_

## Affects
This PR primarily affects the manual gateway and paypal standard gateway. In this solution, the function hooking into 'give_paypal_standard_cc_form' is modified to return a note and image if billing fields are not setup. 

It also introduces the 'give_manual_form_output function, which hooks into 'give_manual_cc_form' to output a note explaining the purpose and utility of test donations.

Beyond this, the PR provides some minor styling improvements to the Sequoia template to accommodate these new gateway notes.

## What to test
Try selecting the test payment gateway, does the note appear correctly? Try the paypal standard gateway without billing enabled - are you shown a note regarding completion of an offsite donation?

## Screenshots:
PRDemo

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by henryholtgeerts** [comment]
> @henryholtgeerts it looks great when testing however we don't want this content to display on the legacy template. Only for the new form template.
> 
> See how it's appearing in the legacy form template as well:
> 
> 

This functionality has been modified, so that it only renders in forms not using the legacy form template. Ref: https://github.com/impress-org/givewp/commit/20c92e562bd15d18bd774fc8974782bf686465e0

**Comment by DevinWalker** [comment]
Thanks - looks great. It's a much better UX when selecting one of these gateways now.

File: includes/gateways/paypal-standard.php [hunk 1]
```diff
@@ -29,7 +32,31 @@ function give_paypal_standard_billing_fields( $form_id ) {
 		return true;
 	}
 
-	return false;
+	if ( getActiveID( getFormId() ) === 'legacy' ) {
```
Comment (ravinderk) [inline includes/gateways/paypal-standard.php:35]: @henryholtgeerts You can use `isLegacyForm` function.

File: includes/gateways/manual.php [hunk 2]
```diff
@@ -14,13 +14,56 @@
...
+
+/**
+ * Manual Gateway form output
+ *
+ * Manual Gateway does not use a CC form, but it does display a note about test donations
+ *
+ * @return bool
+ **/
+function give_manual_form_ouput() {
+	if ( getActiveId( getFormId() ) === 'legacy') {
```
Comment (ravinderk) [inline includes/gateways/manual.php:35]: @henryholtgeerts You can use `isLegacyForm` function.

**Pull request merged**